client/my-sites/plugins/plugins-browser-list/index.jsx

```diff
@@ -39,13 +40,13 @@ module.exports = React.createClass( {
 		return pluginsViewsList;
 	},
 
-	getPlaceholdersViews: function() {
-		return Array.apply( null, Array( this.props.size || this._DEFAULT_PLACEHOLDER_NUMBER ) ).map( function( item, i ) {
+	getPlaceholdersViews() {
+		return Array.apply( null, Array( this.props.size || this._DEFAULT_PLACEHOLDER_NUMBER ) ).map( ( item, i ) => {
 			return <PluginBrowserItem isPlaceholder key={ 'placeholder-plugin-' + i } />;
 		} );
 	},
 
-	getViews: function() {
+	getViews() {
```

Comment:
This could be done a different PR, but I think it would be great rename getViews and getLink to renderViews and renderLink. To be in line with the new guideline.